java/code/src/com/suse/manager/webui/templates/system-common.jade

```diff
@@ -32,4 +32,4 @@ else
     a(href="/rhn/help/reference/en-US/ref.webui.systems.systems.jsp#s3-sm-system-details" target="_blank")
       i.fa.fa-question-circle.spacewalk-help-link()
 
-!{h.renderNavigationMenu(request, "/WEB-INF/nav/system_detail.xml")}
+!{h.renderSystemNavigationMenu(request)}
```

Comment:
Maybe we should use renderNavigationMenuWithParams explicitly with the additional params here. It's easier to follow with eyes, and if we follow this approach we'll have N number of helper methods for each page group.

What do you think?

Reply:
:+1: